### PR TITLE
Added try catch to catch the iframe access origin error for trustly.

### PR DESCRIPTION
**Pull request by virrios**
Fix "Blocked a frame with origin  from accessing a cross-origin frame" for trustly,
showing up in Sentry

## What?
Bugbash Fix To stop showing "Blocked a frame with origin  from accessing a cross-origin frame" in Sentry
and instead catch it in the frontend.

Added try catch to wrap **actualHandleIframeLoad** 

## Why?
These changes were mad because the iframe shows up, and should not bloat the Sentry issue logs.

CatchError

### CODE REVIEW

File: src/client/pages/ConnectPayment/components/TrustlyModal.tsx [hunk 1]
```diff
@@ -94,11 +94,16 @@ export const TrustlyModal: React.FC<Props> = ({
...
-            )(contentWindow)
+
+            try {
+              await actualHandleIframeLoad(
+                setIsOpen,
+                setIsSuccess,
+                generateTrustlyUrl,
+              )(contentWindow)
+            } catch (e) {
+              console.error(e.message)
```
Comment (palmenhq) [inline src/client/pages/ConnectPayment/components/TrustlyModal.tsx:105]: i guess we could a) use just `console.error(e)` right?, and b) won't sentry capture this anyways if we write it on `console.error`? maybe thats ok and we can ignore it in sentry instead

**Review by palmenhq — COMMENTED** [review]
nice one!

File: src/client/pages/ConnectPayment/components/TrustlyModal.tsx (hunk 1 above)
Comment (virrios) [reply src/client/pages/ConnectPayment/components/TrustlyModal.tsx:105]: That's what I wanted to test Sentry locally for ;) . I did not receive and issue when catching the error. (I spammed that connect button like a pro). 
 But if I removed the try/catch i got an issue in Sentry instantly.

We can use console.error(e to  get the whole stacktrace as well but I though the idea was just using this as a reminder and if you just use console.error(e) the output is a bit messy.

**Review by palmenhq — APPROVED** [review]
Awesome!

**Pull request merged**